src/networking/responses/checkout-complete-response.ts

```diff
@@ -1,6 +1,6 @@
 export interface CheckoutCompleteResponse {
   operation_session_id: string;
   gateway_params: {
-    client_secret?: string;
+    client_secret?: string | null;
```

Comment:
Is this needed? it was already `undefined`, right?

Reply:
I think I will leave it as is. I can confirm the backend returns null but not sure if removing the undefined could lead to any cases I am not seeing right now.